docs: example of csharp example serializer (#567)
asyncapi/modelina: examples/csharp-generate-serializer/README.md

> # C# Generate `Generate Serializer`

Reviewer: ```suggestion
# C# Generate serializer functions
```

> A basic example on how to generate models and want to serialize the data to validate against the schema.

Reviewer: ```suggestion
A basic example of how to generate models which includes a way to serialize them into JSON.
```
